src/wazuh_qa_framework/generic_modules/tools/configuration.py

```diff
@@ -0,0 +1,249 @@
+
+    Returns:
+        List of str: List of str with the custom Wazuh configuration.
+    """
+
+    def create_elements(section: ET.Element, elements: List):
+        """
+        Insert new elements in a Wazuh configuration section.
+
+        Args:
+            section (ET.Element): Section where the element will be inserted.
+            elements (list): List with the new elements to be inserted.
+        Returns:
+            ET.ElementTree: Modified Wazuh configuration.
+        """
+        tag = None
+        for element in elements:
+            for tag_name, properties in element.items():
+                tag = ET.SubElement(section, tag_name)
+                new_elements = properties.get('elements')
+                attributes = properties.get('attributes')
+                if attributes is not None:
+                    for attribute in attributes:
+                        if isinstance(attribute, dict):  # noqa: E501
+                            for attr_name, attr_value in attribute.items():
+                                tag.attrib[attr_name] = str(attr_value)
+                if new_elements:
+                    create_elements(tag, new_elements)
+                else:
+                    tag.text = str(properties.get('value'))
+                    attributes = properties.get('attributes')
+                    if attributes:
+                        for attribute in attributes:
+                            if attribute is not None and isinstance(attribute, dict):  # noqa: E501
+                                for attr_name, attr_value in attribute.items():
+                                    tag.attrib[attr_name] = str(attr_value)
+                tag.tail = "\n    "
+        tag.tail = "\n  "
+
+    def purge_multiple_root_elements(str_list: List[str], root_delimeter: str = "</ossec_config>") -> List[str]:
+        """
+        Remove from the list all the lines located after the root element ends.
+
+        This operation is needed before attempting to convert the list to ElementTree because if the ossec.conf had more
+        than one `<ossec_config>` element as root the conversion would fail.
+
+        Args:
+            str_list (list or str): The content of the ossec.conf file in a list of str.
+            root_delimeter (str, optional: The expected string to identify when the first root element ends,
+            by default "</ossec_config>"
+
+        Returns:
+            list of str : The first N lines of the specified str_list until the root_delimeter is found. The rest of
+            the list will be ignored.
+        """
+        line_counter = 0
+        for line in str_list:
+            line_counter += 1
+            if root_delimeter in line:
+                return str_list[0:line_counter]
+        else:
+            return str_list
+
+    def to_elementTree(str_list: List[str], root_delimeter: str = "</ossec_config>") -> ET.ElementTree:
```

Comment:
This function's name is in a mix of snake case and camel case. It is recommended to use only snake case in Python.
```suggestion
    def to_element_tree(str_list: List[str], root_delimeter: str = "</ossec_config>") -> ET.ElementTree:
```

Reply:
Done in 72828b5